tasks/main.yml

```diff
@@ -90,3 +90,5 @@
     - configuration
     - haproxy
     - haproxy-start-enable-service
+
+- meta: flush_handlers
```

Comment:
I rather not have this, since this flushes all open handlers. Is there any reason why we would need it?